docs/EN_US/ContainerizedHPCC/ContainerizedMods/AzureDeployChapter.xml

```diff
@@ -0,0 +1,645 @@
+        single page):</para>
+
+        <programlisting>az aks nodepool add \ 
+--name sysnodepool \ 
+--cluster-name aks-hpcc \ 
+--resource-group rg-hpcc \ 
+--mode System \ 
+--enable-cluster-autoscaler \ 
+--node-count=2 \ 
+--min-count=1 \ 
+--max-count=2 \ 
+--node-taints CriticalAddonsOnly=true:NoSchedule 
+</programlisting>
+
+        <para>Delete the automatically created default pool, which is
+        typically named "nodepool1".</para>
+
+        <para>Once again enter the following command on one line, (without
+        connectors "\" if possible).</para>
+
+        <programlisting>az aks nodepool delete \ 
+--name nodepool1 \ 
+--cluster-name aks-hpcc \ 
+--resource-group rg-hpcc 
+</programlisting>
+
+        <para>Having at least one user node pool is required.</para>
+
+        <para>Next add a <emphasis>user node pool</emphasis> which will
+        schedule the HPCC Systems pods. Once again, do so on a single line
+        without the connectors, if possible:</para>
+
+        <programlisting>az aks nodepool add \ 
+--name usrnodepool1 \ 
+--cluster-name aks-hpcc \ 
+--resource-group rg-hpcc \ 
+--enable-cluster-autoscaler \ 
+--node-count=2 \ 
+--min-count=1 \ 
+--max-count=2 \ 
+--mode User 
+</programlisting>
+
+        <para>For more information about Azure virtual machine pricing and
+        types, please visit <ulink
+        url="https://azure.microsoft.com/en-us/pricing/details/virtual-machines/">https://azure.microsoft.com/en-us/pricing/details/virtual-machines/linux/</ulink></para>
+      </sect3>
+
+      <sect3 id="AZ_Configure_Credentials">
+        <title>Configure Credentials</title>
+
+        <para>Set up Kubernetes so that it can communicate and authenticate
+        within the AKS. This way Kubernetes, <emphasis>kubectl</emphasis>, and
+        Helm can interact with Azure. Configure the Kubernetes client
+        credentials. To do this enter the following command:</para>
+
+        <programlisting>az aks get-credentials --resource-group rg-hpcc --name aks-hpcc --admin</programlisting>
+      </sect3>
+    </sect2>
+
+    <sect2 id="AZ_Installing_TheHelmChrts">
+      <title>Installing the Helm charts</title>
+
+      <para>This section will demonstrate how to fetch, modify, and deploy the
+      HPCC Systems charts. First we will need to access the HPCC Systems
+      repository.</para>
+
+      <para>Add, or update if already installed, the HPCC Systems Helm chart
+      repository:</para>
+
+      <programlisting>helm repo add hpcc https://hpcc-systems.github.io/helm-chart/</programlisting>
+
+      <para>To update the repository:</para>
+
+      <programlisting>helm repo update</programlisting>
+
+      <para>Updating, even if you already installed the HPCC Systems
+      repository, is a good idea.</para>
+
+      <sect3 id="AZ_INstalling-the-HPCCComponents">
+        <title>Installing the HPCC Systems components</title>
+
+        <para>In order for a even a basic installation to succeed, it must
```

Comment:
The section on persistent storage in the book is only for Local DD deployments. Remove this reference until we have details for Azure in this chapter.